api/src/test/java/io/opentelemetry/api/baggage/BaggageUtilsTest.java

```diff
@@ -9,10 +9,16 @@
 
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.Scope;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class BaggageUtilsTest {
 
+  @BeforeEach
+  void setUp() {
+    Context.root().makeCurrent();
```

Comment:
Ditto, don't make the root current, it's the default.

Reply:
yeah, this was my tired brain doing something silly. Interestingly, it did fix up all the tests!